src/utilities.jl

```diff
@@ -79,3 +79,9 @@ linear_scheduler(drop_prob::Nothing; depth::Integer) = fill(drop_prob, depth)
 function _checkconfig(config, configs)
     @assert config in configs "Invalid configuration. Must be one of $(sort(collect(configs)))."
 end
+
+# Utility function to be used for generating random normal distribution used for 
+# initializing layer parameters
+function _random_normal(shape...)
+    return Float64.(rand(Normal(0.0f0, 0.02f0), shape...))
+end
```

Comment:
Why the random_normal initialization specifically? I don't think the paper mentions anything in particular. This adds a dependency on Distributions.jl which didn't exist before I believe.

Reply:
If this doesn't exist in Flux already (I would be surprised if it didn't), then it should be added there and not in Metalhead.

Reply:
Maybe using [`Flux.randn32`](https://fluxml.ai/Flux.jl/dev/utilities/#Flux.randn32)?